tests/test_mlperf_datacenter.py

```
@@ -0,0 +1,396 @@
+        """Test per 8-GPU node normalization calculation."""
+        filter_selections = {
+            "benchmarks": ["llama3.1-8b-datacenter"],
+            "scenarios": ["Offline"],
+        }
+
+        fig = create_normalized_comparison_chart(
+            sample_mlperf_df,
+            "llama3.1-8b-datacenter",
+            "Offline",
+            filter_selections,
+            "Per 8-GPU Node (per-node perf × 8 ÷ GPUs/node)",
```

Ruff (0.14.3)

String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)